scripts/localizationGenerator.js

```diff
@@ -40,7 +40,17 @@ class LocalizationGenerator {
     const localizationObj = featureFolders.reduce((acc, folder) => {
       const parsedLocalizationFromFolder = this.getLocalizationFromFolder(folder);
 
-      return { ...acc, ...parsedLocalizationFromFolder };
+      Object.keys(parsedLocalizationFromFolder).forEach(language => {
```

Comment:
can you explain for what this update is?

Reply:
Yes, of course. In the previous version, we were only getting the localization file for one page because of our object structure: {en: {translation: {...}}, uk: {translation: {...}}}. So, in the loop, the fields en and uk in the accumulator (acc) were being overwritten because they had the same structure as in parsedLocalizationFromFolder variable.